ci_framework/roles/hive/tasks/openstack_cluster_claim.yml

```diff
@@ -0,0 +1,70 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+# Claims a cluster from a the provided pool
+-
+  name: Claim an OpenShift Cluster
+  hosts: localhost
+  vars:
+    claim_name: "{{ cifw_hive_claimName }}"
+    cluster_pool: "{{ cifw_hive_clusterPoolName }}"
+    rtn_kube: "{{ cifw_hive_kubePath }}"
+    life_time: "{{ cifw_hive_lifetime | default(false) }}"
+
+  tasks:
+    - name: Check if ClusterPool exists
+      ansible.builtin.shell:
+        cmd: "{{ oc_cmd }} get ClusterPool {{ cluster_pool }}"
```

Comment:
You may want to define `oc_cmd` variable alternatively, you could do `oc --kubeconfig {{ cifmw_hive_kubeconfig`